package.json

```diff
@@ -1,17 +1,28 @@
 {
   "name": "@protocolink/contract",
-  "license": "GPL-3.0-or-later",
   "version": "0.1.0",
   "description": "Protocolink Contract",
+  "keywords": [
+    "furucombo",
+    "protocolink"
+  ],
+  "repository": "https://github.com/dinngo/protocolink-contract.git",
+  "license": "GPL-3.0-or-later",
   "files": [
-    "src/**/*.sol"
+    "contracts"
```

Comment:
@Jeff-CCH Will the zksync branch use the "contracts" folder later on? My understanding is that if we can use hardhat's `source` config, then we shouldn't need to change "src/" to "contracts/" in the zksync branch

Reply:
Yeah! zkSync branch will use `src/` as the contract folder via `hardhat.config.js`